front/app/modules/commercial/content_builder/admin/components/CraftComponents/Accordion/index.tsx

```diff
@@ -0,0 +1,111 @@
+import React from 'react';
+
+// components
+import QuillEditedContent from 'components/UI/QuillEditedContent';
+import QuillEditor from 'components/UI/QuillEditor';
+
+// craft
+import { useNode } from '@craftjs/core';
+import {
+  Box,
+  Accordion,
+  Title,
+  Input,
+  Label,
+  Toggle,
+} from '@citizenlab/cl2-component-library';
+
+// hooks
+import { useTheme } from 'styled-components';
+
+// intl
+import messages from '../../../messages';
+import { FormattedMessage, injectIntl } from 'utils/cl-intl';
+
+const AccordionSection = ({ text, title, openByDefault }) => {
```

Comment:
This should be named Accordion. I know it conflicts with the import from the component library but you can rename things as you import them:
` import { Accordion as AccordionComponent } from '@citizenlab/cl2-component-library';`

Reply:
Ah perfect! Yes that is much better. I'll update all the names then. Thank you!